docs/ONNX_Runtime_for_Mobile_Platforms.md

```diff
@@ -93,6 +93,10 @@ The follow options can be used to reduce the build size. Enable all options that
     - Whilst the operator kernel reduction script will disable all unused ML operator kernels, additional savings can be achieved by removing support for ML specific types. If you know that your model has no ML ops, or no ML ops that use the Map type, this flag can be provided. 
     - See the specs for the [ONNX ML Operators](https://github.com/onnx/onnx/blob/master/docs/Operators-ml.md) if unsure.
 
+  - Use shared libc++ on Android (`--android_cpp_shared`)
+    - Building using the shared libc++ library instead of the default static libc++ library will result in a smaller libonnxruntime.so library.
+    - See [Android NDK documentation](https://developer.android.com/ndk/guides/cpp-support) for more information.
```

Comment:
Maybe add  something like " libc++ is not a system library. If you use libc++_shared.so, it must be included in your APK." to be more clear?

Reply:
That is covered in the linked doco (https://developer.android.com/ndk/guides/cpp-support#shared_runtimes) so I'm inclined to keep this document reasonably high level and leave it as is.